workloads/virtual-machines/virtualized-hardware-configuration.md

```diff
@@ -278,3 +278,38 @@ Various VM resources, such as a video adapter, IOThreads, and supplementary syst
 
 Please see [how Pods with resource requests are scheduled](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#how-pods-with-resource-requests-are-scheduled) for additional information on resource requests and limits.
 
+## Hugepages
+
+KubeVirt give you possibility to use hugepages as backing memory for your VM. You will need to provide desired size of hugepages to use, for example for x86_64 architecture it can be `2Mi`.
+
+```yaml
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: myvm
+spec:
+  domain:
+    resources:
+      requests:
+        memory: "64Mi"
+    memory:
+      hugepages:
+        pageSize: "2Mi"
```

Comment:
Can you explicitely call out the mentioned fields in the verbal description? `resources.requests.memory` and memory.hugepages.pageSize`?